packages/react-native-session-replay/android/src/main/kotlin/com/datadog/reactnative/sessionreplay/ReactNativeSessionReplayExtensionSupport.kt

```diff
@@ -6,22 +6,55 @@
 
 package com.datadog.reactnative.sessionreplay
 
+import android.annotation.SuppressLint
+import android.util.Log
 import android.view.View
+import androidx.annotation.VisibleForTesting
 import com.datadog.android.sessionreplay.ExtensionSupport
 import com.datadog.android.sessionreplay.SessionReplayPrivacy
 import com.datadog.android.sessionreplay.internal.recorder.OptionSelectorDetector
 import com.datadog.android.sessionreplay.internal.recorder.mapper.WireframeMapper
+import com.datadog.reactnative.sessionreplay.mappers.ReactTextMapper
+import com.datadog.reactnative.sessionreplay.mappers.ReactViewGroupMapper
+import com.facebook.react.bridge.ReactContext
+import com.facebook.react.uimanager.UIManagerModule
+import com.facebook.react.views.text.ReactTextView
+import com.facebook.react.views.textinput.ReactEditText
 import com.facebook.react.views.view.ReactViewGroup
 
-internal class ReactNativeSessionReplayExtensionSupport : ExtensionSupport {
+internal class ReactNativeSessionReplayExtensionSupport(
+    private val reactContext: ReactContext,
+) : ExtensionSupport {
 
     override fun getCustomViewMappers(): Map<SessionReplayPrivacy, Map<Class<*>, WireframeMapper<View, *>>> {
-        return mapOf(SessionReplayPrivacy.ALLOW to mapOf(
-            ReactViewGroup::class.java to ReactViewGroupMapper() as WireframeMapper<View, *>
-        ))
+        val uiManagerModule = getUiManagerModule()
```

Comment:
If it's `null` I believe we should also add some telemetry to help debugging, what do you think?

Reply:
If it won't be too noisy, then yes. My comment here https://github.com/DataDog/dd-sdk-reactnative/pull/569#discussion_r1429754120 should cover it once `InternalLogger` call is added.

Reply:
I think it makes sense to register the mappers, because even in the mappers that rely on it there are additional properties that are derived from the reactbackgrounddrawable without connection to the shadownode